GameData/RP-1/Contracts/Earth Satellites/FirstTundraSat.cfg

```diff
@@ -7,7 +7,7 @@ CONTRACT_TYPE
 	
```

Comment:
I think doing anything with Tundra should wait for Comm App 2, it doesn't fit in Comm App 1.

Reply:
That's the original program, didn't make any changes to it. 

Should we maybe add an exclusion to the new programs so that if someone as already accepted the original Earth Satellites they won't get the new EOS/CA?

Reply:
Except the changes shown here involve changing the description to mention it's from either program (Targeted or CA1), the requirements now have an ANY for either of those programs (and an ANY for its prereq contract) and some textual changes. I would have expected there to be no changes in the file since the old program has all its own contracts untouched, so no changes are needed here to check for completions of the other contracts.

You're very right however that EOS and CA should not be allowed if you've already taken Targeted, and Targeted or Comm Net respectively. Don't worry about that part however, since I have to do that in code (and in a somewhat ugly way, at that!)

Reply:
> Except the changes shown here involve changing the description to mention it's from either program (Targeted or CA1), the requirements now have an ANY for either of those programs (and an ANY for its prereq contract) and some textual changes. I would have expected there to be no changes in the file since the old program has all its own contracts untouched, so no changes are needed here to check for completions of the other contracts.


Oh yeah that.